enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -82,7 +83,7 @@ export function useLoadQuestion({
   // Avoid re-running the query if the parameters haven't changed.
   const sqlParameterKey = getParameterDependencyKey(initialSqlParameters);
 
-  const shouldLoadQuestion = cardId != null;
+  const shouldLoadQuestion = cardId != null || deserializedCard != null;
```

Comment:
`cardId` checks if we should default the loading state when rendering the interactive question component directly. And `deserializedCard` checks whether we need to do the same when drilling from either an `EditableDashboard` or an `InteractiveDashboard`